book.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'item'
+
+class Book < Item
+  attr_accessor :publisher, :cover_state
+
+  def initialize(publisher, cover_state)
+    super()
```

Comment:
- [ ] Please add the publish_date in the book initialization and also in the super. This is because the book class inherited the Item class.
- Hint:
```suggestion
  def initialize(publisher, cover_state, publish_date)
    super(publish_date)
```

Reply:
I have implemented it . Thank you @Johnadibe